workspaces/liferay-sample-workspace/client-extensions/liferay-sample-custom-element-3/src/app/app.component.spec.ts

```diff
@@ -34,4 +34,14 @@ describe('AppComponent', () => {
 			'liferay-sample-custom-element-3 app is running!'
 		);
 	});
+
+	it('should run ngOnInit once', () => {
```

Comment:
Not sure if we want this kind of tests in examples since they are so specific to our stuff, not to the example itself...